monkey/common/agent_registration_data.py

```diff
@@ -17,6 +17,7 @@ class AgentRegistrationData(InfectionMonkeyBaseModel):
     parent_id: Optional[UUID]
     cc_server: SocketAddress
     network_interfaces: Sequence[IPv4Interface]
+    sha256: str
```

Comment:
We should add some validation to this. A validator that checks this against a regex should be sufficient, like `r"[0-9a-f]{64}"`